tests/tensor_ext/simd_pack.mlir

```diff
@@ -0,0 +1,15 @@
+// RUN: heir-opt --secretize=entry-function=add --wrap-generic --canonicalize --cse \
+// RUN:   --simd-pack %s | FileCheck %s
+
+module  {
+  // CHECK-LABEL: @add
+  // CHECK-SAME: %[[arg0:.*]]: !secret.secret<[[ty:tensor<1024xi16, .*>]]>, %[[arg1:.*]]: !secret.secret<[[ty]]>
```

Comment:
You should assert the attribute values are correctly generated.

Reply:
I always had a hard time matching on attributes (something about the brackets in the struct? not sure), but TIL about `CHECK{LITERAL}`